### DISCUSSION TITLE
Tests with URL RPC Arbitrum Sepolia fail at Foundry Fund me contract

### DISCUSSION THREAD

**Discussion by danilomarcus** [Cyfrin/foundry-full-course-cu#1469] [Q&A] [answered]
Hello guys! Does anyone know why my tests is not working with Arbitrum SEPOLIA

I did the same API configuration on Alchemy, it works for SEPOLIA ETH and Mainnet ETH.

Here my command:

`forge test -vv --fork-url $ARB_SEPOLIA_RPC_URL`

And here the error returned:

```
2024-03-06T03:13:52.742705Z ERROR forge::runner: setUp failed reason=EvmError: Revert contract=0x7FA9385bE102ac3EAc297483Dd6233D62b3e1496

Running 1 test for test/FundMeTest.t.sol:FundMeTest
[FAIL. Reason: setup failed: EvmError: Revert] setUp() (gas: 0)
Test result: FAILED. 0 passed; 1 failed; 0 skipped; finished in 6.32s

Ran 1 test suites: 0 tests passed, 1 failed, 0 skipped (1 total tests)

Failing tests:
Encountered 1 failing test in test/FundMeTest.t.sol:FundMeTest
[FAIL. Reason: setup failed: EvmError: Revert] setUp() (gas: 0)
```
Thanks in advance!

**Comment by EngrPips** [comment, marked as answer]
Hello @danilomarcus , I never tried to run the test on Arbitrum SEPOLIA myself but seeing that your test failed at the set up level, first thing i would think is did you use the correct feed price address for the test.

**Reply by danilomarcus** [reply]
You`re right, I got the wrong PriceFeed!

Thank you so much!

**Reply by EngrPips** [reply]
Anytime buddy, i am glad i was able to help.
